utils/prot16_to_iterm2.rb

```diff
@@ -0,0 +1,177 @@
+  'Badge Color'         => 'red',
+  'Bold Color'          => 'red',
+  'Cursor Color'        => 'dhl',
+  'Cursor Guide Color'  => 'dt2',
+  'Cursor Text Color'   => 'dt2',
+  'Foreground Color'    => 'dt1',
+  'Link Color'          => 'blu',
+  'Selected Text Color' => 'dt1',
+  'Selection Color'     => 'dhl'
+}.freeze
+
+# Color class, responsible for representation and conversion of colors
+## CONSTRUCTORS
+# from_rgb_zero_one(red, green, blue)
+#   red: a value in the range (0..1) for the red Component
+#   green: a value in the range (0..1) for the green Component
+#   blue: a value in the range (0..1) for the blue Component
+# from_hex_string(hex)
+#   hex: a string in the format #RRGGBB or RRGGBB.
+## PRESENTERS / ACCESSORS
+# red: a value in the 0..1 range for the red component
+# green: a value in the range (0..1) for the green Component
+# blue: a value in the range (0..1) for the blue Component
+class Color
+  attr_accessor :red
+  attr_accessor :green
+  attr_accessor :blue
+
+  def self.from_rgb_zero_one(red, green, blue)
+    new(red, green, blue)
+  end
+
+  def self.from_hex_string(hex)
+    matches = hex.match(/#?(..)(..)(..)/)
+    red   = matches[1].hex / 255.0
+    green = matches[2].hex / 255.0
+    blue  = matches[3].hex / 255.0
+    new(red, green, blue)
+  end
+
+  private
+
+  def initialize(red, green, blue)
+    @red = red
+    @green = green
+    @blue = blue
+  end
+
+  def to_48bit_hex
+    red_part   = sprintf("%04x", (red.to_f   * 65535).round)
+    green_part = sprintf("%04x", (green.to_f * 65535).round)
+    blue_part  = sprintf("%04x", (blue.to_f  * 65535).round)
+    "##{red_part}#{green_part}#{blue_part}"
+  end
+
+  def to_24bit_hex
+    red_part   = sprintf("%02x", (red.to_f   * 255).round)
+    green_part = sprintf("%02x", (green.to_f * 255).round)
+    blue_part  = sprintf("%02x", (blue.to_f  * 255).round)
+    "##{red_part}#{green_part}#{blue_part}"
+  end
+end
+
+def iterm_color_block(context)
+  b = binding
+  ERB.new("
+  <key><%= context[:label] %></key>
+  	<dict>
+  		<key>Alpha Component</key>
+  		<real>1</real>
+  		<key>Blue Component</key>
+  		<real><%= context[:color][:blue] %></real>
+  		<key>Color Space</key>
+  		<string>sRGB</string>
+  		<key>Green Component</key>
+  		<real><%= context[:color][:green] %></real>
+  		<key>Red Component</key>
+  		<real><%= context[:color][:red] %></real>
+  	</dict>
+".gsub(/^\s+/, ""), 0, "").result b
+end
+
+FILE_HEADER = '<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>'.freeze
+
+FILE_FOOTER = '</dict>
+</plist>'.freeze
+
+scheme_names = %w(alto archaic bionis blau caprice cyprium equinox ficus flowerbed gaia magus nefelio neptune ocarina oliveira orionis overgrowth playa seabed sonho symbiosis vin)
+scheme_names.each do |scheme_name|
+  input_file = "#{scheme_name}/lb/#{scheme_name}-values.yml"
```

Comment:
I was in the process of cleaning up the repo. I thus changed this to #{scheme_name}/general/#{schemme_name}-values.yml